pkgs/applications/misc/holochain-launcher-bin/default.nix

```diff
@@ -0,0 +1,57 @@
+{ stdenv
+, lib
+, fetchurl
+, autoPatchelfHook
+, pkg-config
+, dpkg
+, openssl
+, webkitgtk
+, libappindicator
+, wrapGAppsHook
+}:
+
+let
+  version = "0.5.1";
+in
+
+stdenv.mkDerivation {
+  name = "holochain-launcher-bin";
```

Comment:
```suggestion
  name = "holochain-launcher";
```
please indicate that through meta.source*

Reply:
please let me know how that is indicated